UnitsNet/CustomCode/UnitClasses/Angle.extra.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright © 2007 by Initial Force AS.  All rights reserved.
```

Comment:
Not your fault, but these headers are outdated. I'll address that myself after merging this in.